src/main/java/edu/iu/uits/mail/MailSigner.java

```diff
@@ -94,6 +103,21 @@ public MailSigner(Properties properties) {
         }
     }
 
+    public String getEmailPassword(String alias) {
```

Comment:
This should be switched to being `protected` instead of public.  Your test should still be able to invoke it if it is placed in the correct package.